src/AudioSettings.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace AudioSettings {
```

Comment:
Just putting this in a namespace make these values global. It'd be better to make this a struct that get explicitly instantiated somewhere.

Reply:
Going to ping @ammunoz here about his thoughts on this too, since he came up with the namespace idea during pair programming. If we plan on storing anything more complicated than a map then I agree that a struct would probably be better.